### PR TITLE
style: formatted non-code files using resp. language standards

### PR DESCRIPTION
**Pull request by OutdatedGuy**
## Description

Formatted all non-code files (like `.md`, `.gitignore`, `.yaml`, etc.) using their respective language formatting standards.

## Type of Change

- [ ] ✨ `feat` -- New feature (non-breaking change which adds functionality)
- [ ] 🛠️ `fix` -- Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ `!` -- Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 `refactor` -- Code refactor
- [ ] ✅ `ci` -- Build configuration change
- [ ] 📝 `docs` -- Documentation
- [ ] 🗑️ `chore` -- Chore

### CODE REVIEW

**Review by spydon — CHANGES_REQUESTED** [review]
@OutdatedGuy the changelogs should not be modified, the lines in the changelog should only be bound to their release commit.

File: packages/melos/pubspec.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ repository: https://github.com/invertase/melos/tree/main/packages/melos
 issue_tracker: https://github.com/invertase/melos/issues
 
 environment:
-  sdk: '>=2.18.0 <3.0.0'
+  sdk: ">=2.18.0 <3.0.0"
```
Comment (spydon) [inline packages/melos/pubspec.yaml:12]: The standard seems to be that these ranges should be with single quotes, check the output of a newly created flutter project for example.
So all pubspec changes should be removed from this PR too.

**Comment by spydon** [comment]
> How does one fix any errors or formatting issues in Changelog file then?

You don't, and also these are not errors, they will parse just fine as markdown. There isn't only one markdown formatting standard, so it depends on what markdown formatting rules you use on how it should be formatted.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -96,4 +96,3 @@ guide.
 
 Please also enable **“Allow edits by maintainers”**, this will help to speed-up the review
 process as well.
-
```
Comment (spydon) [inline CONTRIBUTING.md:99]: The last line should be empty, like it was

**Review by spydon — COMMENTED** [review]
Just found one more thing, then this is good to be merged

The pubspec description also now is longer than 80 chars, but I don't think that matters seeing that there are other lines in that file going past 80 too.

File: CONTRIBUTING.md (hunk 2 above)
Comment (spydon) [reply CONTRIBUTING.md:99]: This is still wrong, the last part of the file should just be reverted to how it was

Comment (OutdatedGuy) [reply CONTRIBUTING.md:99]: @spydon do we want one empty line in the end or two?

Cuz generally the files ends with one empty line (i.e. new line) and Github diff preview does not show the last new line. But the file originally had two empty new lines at end and I removed the extra empty/new line.

Comment (OutdatedGuy) [reply CONTRIBUTING.md:99]: Screenshot 2023-12-22 at 8 12 30 PM

Here the left file has no empty/new line at the end and the right one have 1 empty/new line at the end. But as you can see in the image above, Github does not preview the last empty line, just the `-` in red circle if new line does not exist at the end of file.

Comment (spydon) [reply CONTRIBUTING.md:99]: Aah right, it should just be one.

**Review by spydon — APPROVED** [review]

**Comment by spydon** [comment]
@OutdatedGuy why did you close it? It was ready to merge :)

**Comment by OutdatedGuy** [comment]
This was not intended, deleting repo didn't used to close a PR 🤔

**Comment by spydon** [comment]
> This was not intended, deleting repo didn't used to close a PR 🤔

Since your PR branch lived on your forked repo it will be deleted if you delete the repository.

**Pull request closed without merging**